### PR TITLE
example config for dokuwiki added

### PR DESCRIPTION
**Pull request by Hobbabobba**

### CODE REVIEW

File: example_configs/dokuwiki.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+```
+<?php
+/**
+ * Protected settings
+ * LDAP configuration example
+ */
+$conf['useacl']         = 1;           //enable ACL
+$conf['authtype']       = 'authldap';  //enable this Auth plugin
+
+$conf['plugin']['authldap']['server']      = 'ldap://192.168.0.2:3890'; #IP of your lldap
```
Comment (nitnelave) [inline example_configs/dokuwiki.md:14]: I would rather have the domain name than the IP here, so something like `ldap://lldap_server:3890`

File: example_configs/dokuwiki.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+$conf['plugin']['authldap']['usertree']    = 'ou=people,dc=example,dc=com';
+$conf['plugin']['authldap']['grouptree']   = 'ou=groups, dc=example, dc=com';
+$conf['plugin']['authldap']['userfilter']  = '(&(uid=%{user})(objectClass=person))';
+$conf['plugin']['authldap']['groupfilter'] = '(&(objectClass=group)(memberUID=member))';
+$conf['plugin']['authldap']['attributes']  = array('cn', 'displayname', 'mail', 'givenname', 'objectclass', 'sn', 'uid', 'memberof');
+ 
+# This is optional but may be required for your server:
```
Comment (nitnelave) [inline example_configs/dokuwiki.md:21]: You can remove this comment.

File: example_configs/dokuwiki.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+$conf['plugin']['authldap']['version']    = 3;
+
+# Optional bind user and password if anonymous bind is not allowed
```
Comment (nitnelave) [inline example_configs/dokuwiki.md:24]: Same here

**Review by nitnelave — CHANGES_REQUESTED** [review]
Looks great! A couple of nitpicks but otherwise it's good. Thanks!

File: example_configs/dokuwiki.md (hunk 1 above)
Comment (Hobbabobba) [reply example_configs/dokuwiki.md]: fixed

**Review by nitnelave — APPROVED** [review]

**Pull request merged**